components/script/dom/webglframebuffer.rs

```diff
@@ -21,7 +21,7 @@ use webrender_traits::{WebGLCommand, WebGLFramebufferBindingRequest, WebGLFrameb
 #[derive(JSTraceable, Clone, HeapSizeOf)]
 enum WebGLFramebufferAttachment {
     Renderbuffer(JS<WebGLRenderbuffer>),
-    Texture(JS<WebGLTexture>),
+    Texture(JS<WebGLTexture>, i32),
```

Comment:
Oh, nice.  I thought the JS<> in there meant the struct would be harder than just doing that.